shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/jaxb/domain/SQLParserTestCases.java

```diff
@@ -203,6 +205,9 @@
 
     @XmlElement(name = "show-index")
     private final List<ShowIndexStatementTestCase> showIndexTestCases = new LinkedList<>();
+
+    @XmlElement(name = "set-variable")
+    private final List<SetVariableStatementTestCase> setVariableStatementTestCases = new LinkedList<>();
```

Comment:
Please do not interrupt `showXXXTestCases`, please move `setVariableStatementTestCases` behind `showTestCases`